### ISSUE TITLE
Error during training Mask RCNN

### ISSUE THREAD

**Issue by EtienneDavid** [open-mmlab/mmdetection#378]
Hello,

First thank you for the amazing toolbox.

I was running a small test with the same train and test set for mask rcnn.
All images are 1000 x 1000
I've let all basic default

But the mAP is 0.04 for box and 0 for segmentation. I produced boxes but without any sense. 

I have only one class so i set n class to 2

What could be the reason of such bad result ?

Best

**Comment by hellock** [comment]
This kind of problem is hard for discussion. You may need to first figure out a more detailed or specific problem by debugging. There can be many reasons for poor performance on a new dataset, including annotation format, dataset size, hyper-parameters tuning, etc.
